### PR TITLE
[CHK-3316] set last used method saving timeout to 0

### PR DESCRIPTION
**Pull request by ciuffagianluca**
Depends on https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/346
Depends on https://github.com/pagopa/pagopa-ecommerce-local/pull/176

#### List of Changes

Rename application properties and yaml file to better address variable meaning

#### Motivation and Context

To implement near real time saving last used method

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/pagopa/transactions/commands/handlers/v2/TransactionRequestAuthorizationHandler.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ public TransactionRequestAuthorizationHandler(
                 "walletAsyncQueueClient"
             ) Optional<WalletAsyncQueueClient> walletAsyncQueueClient,
             @Value("${azurestorage.queues.transientQueues.ttlSeconds}") Integer transientQueuesTTLSeconds,
-            @Value("${npg.authorization.request.timeout.seconds}") Integer npgAuthRequestTimeout,
+            @Value("${authorization.event.visibilityTimeoutSeconds}") Integer saveLastUsedMethodTimeout,
```
Comment (scaminati-bv) [inline src/main/java/it/pagopa/transactions/commands/handlers/v2/TransactionRequestAuthorizationHandler.java:98]: If the first step of event AuthRequest changes in the event-dispatcher this variable name (_saveLastUsedMethodTimeout_) may give incorrect information. 
Might it make sense to leave a more generic name for the timeout? (e.g. _AuthRequestEventTimout_)

Comment (pietro-tota) [reply src/main/java/it/pagopa/transactions/commands/handlers/v2/TransactionRequestAuthorizationHandler.java:98]: yes, miss this during refactoring made [here](https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/583/commits/dfe461467bfa0490972a9137341cf1a9bf67a38e) 😄 fixing it, good catch

Comment (pietro-tota) [reply src/main/java/it/pagopa/transactions/commands/handlers/v2/TransactionRequestAuthorizationHandler.java]: fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/583/commits/da90b681b3e4db2f6d9357c18c9ce25692e3c9d1

File: .devops/code-review-pipelines.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local 
-      ref: main
+      ref: CHK-3316-align-env
```
Comment (pietro-tota) [inline .devops/code-review-pipelines.yml:15]:
```suggestion
      ref: main
```

TODO: to be restored after dependent ecommerce-local PR merge

**Review by scaminati-bv — APPROVED** [review]

**Review by CianoDanilo — APPROVED** [review]

**Pull request merged**